documentation/modules/cruise-control/proc-executing-cruise-control-rebalance-operations.adoc

```diff
@@ -0,0 +1,9 @@
+// This module is included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-executing-cruise-control-rebalance-operations-{context}']
+= Executing a cluster rebalance
+
+TBD
```

Comment:
Should these be deleted? Or at least comment out everything and keep the file so that the title and the TBD doesn't render?

Reply:
Yes, good spot. Please comment out the include statements for these modules.

```
//include::../../modules/cruise-control/proc-interacting-with-cruise-control-api.adoc[leveloffset=+1]

//include::../../modules/cruise-control/proc-executing-cruise-control-rebalance-operations.adoc[leveloffset=+1]
```

Reply:
Commented these out in `assembly-cruise-control-concepts.adoc` so they don't show up in the docs